python/pyiceberg/io/__init__.py

```diff
@@ -55,6 +55,17 @@
 HDFS_PORT = "hdfs.port"
 HDFS_USER = "hdfs.user"
 HDFS_KERB_TICKET = "hdfs.kerberos_ticket"
+GCS_TOKEN = "gcs.oauth2.token"
+GCS_TOKEN_EXPIRES_AT = "gcs.oauth2.token-expires-at"
+GCS_PROJECT_ID = "gcs.project-id"
+GCS_ACCESS = "gcs.access"
+GCS_CONSISTENCY = "gcs.consistency"
+GCS_CACHE_TIMEOUT = "gcs.cache-timeout"
+GCS_REQUESTER_PAYS = "gcs.requester-pays"
+GCS_SESSION_KWARGS = "gcs.session-kwargs"
+GCS_ENDPOINT_URL = "gcs.endpoint-url"
+GCS_DEFAULT_LOCATION = "gcs.default-bucket-location"
```

Comment:
What does this control? Does it modify URIs to fill in a bucket if it isn't present? I'm not sure that's a good idea.